zap/src/main/java/org/parosproxy/paros/core/scanner/VariantCookie.java

```diff
@@ -121,7 +123,7 @@ public List<NameValuePair> getParamList() {
     @Override
     public String setParameter(
             HttpMessage msg, NameValuePair originalPair, String name, String value) {
-        return setParameter(msg, originalPair, name, value, false);
+        return setParameter(msg, originalPair, name, value, !encodeCookieValues);
```

Comment:
This negative logic bothers me. Above we set the default as false so won't !false be true? (In the GUI won't the checkbox be unchecked if the default is false?)

Reply:
Will be changing this as part of a change suggested by @thc202 in irc 😉